src/app/dashboard/sensor-widget/sensor-widget.template.scss

```diff
@@ -1,15 +1,82 @@
-.sensor-widget_statistic-button {
+$value-color: #f3ac2e;
+$widget-padding: 27px;
+$widget-width: 182px;
+
+.sensor-widget {
     position: relative;
-    display: block;
-    float: left;
-    cursor: pointer;
-    width: 24px;
-    height: 26px;
-    margin-top: 1px;
-    background-color: #dedede;
-    background-image: url(../assets/statistic.svg);
-
-    &:hover {
-        box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5);
+    width: $widget-width;
+    height: $widget-width;
+    padding: 2px;
+    font-family: Roboto;
+    text-align: center;
+    color: white;
+    background-color: #169eb1;
+    border: solid 1px white;
+    overflow: hidden;
+
+    &_footer, &_header {
+        position: absolute;
+        width: 100%;
+        bottom: 0;
+        height: $widget-padding;
```

Comment:
It's not clear why you use $widget-padding variable for height, max-height and other css rules... (the same for $widget-width).
Should we use separate variables for them?